### ISSUE TITLE
0.17 betas fail to install

### ISSUE THREAD

**Issue by bochecha** [learningequality/ka-lite#5353]
## Summary

Here's what I get:

```
Collecting django-tastypie==0.12.2dev0 (from ka-lite==0.17.0b1)
  Could not find a version that satisfies the requirement django-tastypie==0.12.2dev0 (from ka-lite==0.17.0b1) (from versions: 0.9.6, 0.9.7, 0.9.9, 0.9.10, 0.9.11, 0.9.12, 0.9.13, 0.9.14, 0.9.15, 0.9.16, 0.10.0, 0.11.0, 0.11.1, 0.12.0, 0.12.1, 0.12.2, 0.13.0, 0.13.1, 0.13.2, 0.13.3)
No matching distribution found for django-tastypie==0.12.2dev0 (from ka-lite==0.17.0b1)
```

Note that I get the same thing with 0.17.0b2 as well.

So I guess they removed their 0.12.2dev0 release from pypi, and you should bump to 0.12.2 (or even later?)

**Comment by benjaoming** [comment]
It's a problem with fetching a special branch of tastypie: #5319 

However, you could try with our version with dependencies bundled: `pip install ka-lite-static --pre` -- does that solve it?

**Comment by benjaoming** [comment]
Fixed in #5373 - will be included in today's prerelease.
